.travis.yml

```diff
@@ -3,8 +3,9 @@ env:
 - TRAVIS=1
 os: 
  - linux
- - osx
-lein: clojure
+# Clojure is not supported on osx
```

Comment:
The page says "Clojure builds are not available on the OSX environment." Maybe lein and be fetched separately?